server/src/main/java/org/elasticsearch/cluster/metadata/MetadataMappingService.java

```diff
@@ -198,7 +198,7 @@ private boolean refreshIndexMapping(IndexService indexService, IndexMetadata.Bui
      */
     public void refreshMapping(final String index, final String indexUUID) {
         final RefreshTask refreshTask = new RefreshTask(index, indexUUID);
-        clusterService.submitStateUpdateTask("refresh-mapping",
+        clusterService.submitStateUpdateTask(String.format("refresh-mapping [%s/%s]", index, indexUUID),
```

Comment:
Let's add only the index name to the task name. Also, we can just concatenate strings (i.e., `"refresh-mapping[" + index + "]"`); no need to use String.format.